README.md

```diff
@@ -26,65 +32,52 @@ Extensive documentation is available [here](http://community.optimism.io/docs/)
 * [`ops`](./ops): Contains Dockerfiles for containerizing each service involved in the protocol,
 as well as a docker-compose file for bringing up local testnets easily
 
-## Quickstart
+## Development Quick Start
 
-### Installation
+### Setup
 
-Dependency management is done using `yarn`.
+Clone the repository, open it, and install dependencies:
 
 ```bash
 git clone git@github.com:ethereum-optimism/optimism.git
 cd optimism
-yarn
+yarn install
 ```
 
-After installing the dependencies, you must also build them so that the typescript
-is compiled down to javascript:
+### Building the TypeScript packages
 
-```bash
-yarn build
-```
-
-When changing branches, be sure to clean the repo before building.
+To build all of the [TypeScript packages](./packages), run:
 
 ```bash
 yarn clean
+yarn build
 ```
 
-### Unit tests
-
-All tests are run in parallel using `lerna`:
-
-```bash
-yarn test
-```
-
-When you want to run tests only for packages that have changed since `master` (or any other branch)
-you can run `yarn lerna run test --parallel --since master`
-
-### Integration Tests
+Packages compiled when on one branch may not be compatible with packages on a different branch.
+**You should recompile all packages whenever you move from one branch to another.**
+Use the above commands to recompile the packages.
 
-#### Running the integration tests
+### Building the rest of the system
 
-The integration tests first require bringing up the Optimism stack. This is done via
-a Docker Compose network. For better performance, we also recommend enabling Docker
-BuildKit
+If you want to run an Optimistic Ethereum node OR **if you want to run the integration tests**, you'll need to build the rest of the system.
 
-```bash
+```
 cd ops
-export COMPOSE_DOCKER_CLI_BUILD=1
-export DOCKER_BUILDKIT=1
-docker-compose build
-cd ../integration-tests
-yarn build:integration
-yarn test:integration
+docker-compose build --parallel
```

Comment:
We should still mention the environment variables, I think they are on by default for newer versions of docker-for-mac but they are not on by default for docker on linux (my version at least) and its significantly faster using them